available_images.md

```diff
@@ -241,14 +241,6 @@ Neuron Containers
 |MXNet 1.8.0    |mx_neuron      |Neuron 1.18.0      |inference 	|inf1   |3.7 (py37) 	        |763104351884.dkr.ecr.us-west-2.amazonaws.com/mxnet-inference-neuron:1.8.0-neuron-py37-sdk1.18.0-ubuntu18.04          |
 
 
```

Comment:
Line 28 at the top of this file also starts a column that references EIA containers. Should this be removed as well?

Reply:
yes, remove them in the new commit